File: 03.business-os/PocketSecretary/android/app/src/main/java/com/lsam/PocketSecretary/commonosconsumer/BusinessOSCommonOSConsumerRegistry.java
package com.lsam.PocketSecretary.commonosconsumer;

public final class BusinessOSCommonOSConsumerRegistry {
    public BusinessOSCommonOSConsumerDescriptor createDefault() {
        return new BusinessOSCommonOSConsumerDescriptor(
                "PocketSecretary",
                "shared_provider",
                "os_side_consumer",
                "~/03.civilization-development/03.business-os/_commonos",
                "CommonOS",
                "PocketSecretary",
                "offline-first"
        );
    }
}

File: 03.business-os/PocketSecretary/iphone/PocketSecretary/BusinessOSCommonOSConsumer/BusinessOSCommonOSConsumerRegistry.swift
import Foundation

enum BusinessOSCommonOSConsumerRegistry {
    static func createDefault() -> BusinessOSCommonOSConsumerDescriptor {
        BusinessOSCommonOSConsumerDescriptor(
            appName: "PocketSecretary",
            providerRole: "shared_provider",
            consumerRole: "os_side_consumer",
            consumerRoot: "~/03.civilization-development/03.business-os/_commonos",
            uiOwner: "CommonOS",
            businessOwner: "PocketSecretary",
            syncMode: "offline-first"
        )
    }
}
